tools/src/test/python/dlpx/virtualization/_internal/test_file_util.py

```diff
@@ -3,6 +3,7 @@
 #
 
 import os
+import ntpath
```

Comment:
maybe im missing it but it doesn't look like we're using ntpath as an import anymore?